mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java

```diff
@@ -288,16 +289,14 @@ private Property toProperty(UDTValue udtValue) {
         return new Property(udtValue.getString(Properties.NAMESPACE), udtValue.getString(Properties.NAME), udtValue.getString(Properties.VALUE));
     }
 
-    private Stream<MessageAttachmentRepresentation> getAttachments(Row row, FetchType fetchType) {
-        switch (fetchType) {
```

Comment:
is this switch/case removal related  to this commit? I can figure out why

Reply:
MessageAttachmentRepresentation only contains metadata of the attachments, which are now always required to position hasAttachment metadata. Hence this switch case removal.